src/main/kotlin/io/inventi/eventstore/eventhandler/model/EventType.kt

```diff
@@ -0,0 +1,10 @@
+package io.inventi.eventstore.eventhandler.model
+
+enum class EventType(val type: String) {
+    V2_INVOICE("v2.Invoice"),
```

Comment:
Why does eventstore-tools need to know about sme project? This does not belong here at all.